zircon.core/src/commonMain/kotlin/org/hexworks/zircon/internal/uievent/impl/UIEventToComponentDispatcher.kt

```diff
@@ -144,7 +144,7 @@ class UIEventToComponentDispatcher(
             )
         )
         lastMousePosition = event.position
-        root.fetchComponentByPosition(lastMousePosition).map {
+        root.fetchComponentByPositionOrNull(lastMousePosition)?.let {
             lastHoveredComponent = it
```

Comment:
what's motivating the decision to move from Maybe to nulls in internal code?

Reply:
Soon [Arrow](https://arrow-kt.io/) will become multiplatform and we planned a migration to Arrow when that happens. At that point `Maybe` becomes obsolete and we'll use `Option` instead. I just can't add the `Option` version right now, as Arrow is not there yet (or is it?).